app/src/main/java/ch/epfl/sweng/hostme/wallet/ExpirationDatePicker.java

```diff
@@ -166,57 +169,85 @@ private void createNotificationChannel() {
     }
 
 
-    private void scheduleNotification(int dayOfMonth, int month, int year) {
+    private void clearPreviouslyScheduledNotif() {
 
+        int all_reminders_size = RemindersDate.values().length;

```

Comment:
I believe the only place we use snake_cased variables is for UI components accessed via R.id.<component_id>. Consider camelCase e.g. `allRemindersSize` or let me know if I'm simply wrong.

Reply:
Thanks for noticing that, I'll change it!